CHANGES.rst

```diff
@@ -6,6 +6,8 @@
 
 - add port_range to wsgi_tests (mrmin)
 
+- Use notSubmittable instead of calculatedProperty to indicate properties that may not be submitted.
```

Comment:
The last release was version 0.30.  I assume this should be on '0.31' but we don't have any release notes between .24 and .30.  I guess we should add version 0.31 to the changes in the buildout.cfg PR. Then tack this PR change below it in here.

Reply:
Where are you seeing that 0.30 release? It's not on PyPI: https://pypi.python.org/simple/snovault/

Reply:
I do see a 0.30 tag, but not a release on PyPI. @hitz @Bek did someone make a tag without updating the changelog or publishing it to PyPI?